homework_11/homework_11.py

```diff
@@ -0,0 +1,96 @@
+# 1 Написати власний декоратор, задачею якого має бути друк назви функції і
+# часу, коли вона була викликана. Декоратор має працювати для різних функцій однаково.
+
+import time
+
+def deco(func):
+    def wrap(*args, **kwargs):
+        str_name = func.__name__
+        print(f"function name - {str_name}")
+        str_time = time.ctime()
+        print(f"start function time - {str_time}")
+        rez = func(*args, **kwargs) + 3
```

Comment:
Remove here `+ 3` - your decorator shouldn't change the result of a function but log the time is was called